### DISCUSSION TITLE
What backend database is being used?

### DISCUSSION THREAD

**Discussion by pieterhumphrey** [realworld-apps/realworld#1345] [Q&A] [answered]
I've looked thru some of the repos looking for what database is being used, but clearly I'm looking in the wrong place.
Some of the discussions mentions Supabase and also Postgres/Heroku.

**Comment by pieterhumphrey** [comment, marked as answer]
@geromegrignon

**Reply by geromegrignon** [reply]
Hi Pieter,

For the public API (api.realworld.io) :

The NodeJS APi is hosted on Heroku.
I used it for a while for the database too but moved when Heroku stopped free tier.
I now use a PostgreSQL database with Supabase.

For the backends implementations, i don't have any listing about what is being used (most backend implementations are not deployed to avoid costs).

**Reply by pieterhumphrey** [reply]
Got it.  Thanks for responding!  for the project in general, not the public demo site (api.realworld.io)
are other backend database configurations something that can be contributed/would be welcome?

**Reply by geromegrignon** [reply]
In this discussion panel you should find ongoing implementation work in progress.
You can contact maintainers to know if they need help.
